psql2mysql/__init__.py

```diff
@@ -98,6 +99,15 @@ def _query_long_text_rows(self, table, textColumns):
         rows = q.execute()
         return rows
 
+    # For some cases MySQL is expecting LONGTEXT instead of TEXT.
+    # We do not need to report such values as too long.
+    def longtextExpected(self, table_name, column):
+        if (self.db_path == "/ironic" and table_name == "nodes"
```

Comment:
Is it possible to configure ironic with different db name?

Reply:
Good question. I'm quite sure we do not allow that in SOC. It's possible some other deployer does it but what would be the point?

In such case this check would obviously not catch the case. I'm not sure if it makes sense to have a special argument just for this (IMO) corner case.

Reply:
Unfortunately (for this case) we allow this: https://github.com/crowbar/crowbar-openstack/blob/master/chef/data_bags/crowbar/template-ironic.json#L37

Reply:
Oh, you got me. 
However, I still think it's a corner case, or is it not? Why would people do it?

What would be the alternative solution? As I wanted this to work without providing the info about target database, it seems the only other way is the extra config option for the tool....